forge-gui/res/cardsfolder/upcoming/grim_giganotosaurus.txt

```diff
@@ -0,0 +1,10 @@
+Name:Grim Giganotosaurus
+ManaCost:5 B G
+Types:Creature Dinosaur
+PT:10/10
+K:Monstrosity:10:10 B G:X
```

Comment:
@Northmoc 
can you fix the X description? 🙏

Reply:
![image](https://github.com/Card-Forge/forge/assets/67333662/51dcd75a-cad6-40a0-b581-962284485dce)
I think I got it fixed myself

Reply:
You're still showing the fresh view that's only based on Oracle
Here's the parsed version:
![image](https://github.com/Card-Forge/forge/assets/8506892/6e5f8d9f-c398-48da-a7c9-4973670a5db0)

Reply:
I think you fixed it ? 
![image](https://github.com/Card-Forge/forge/assets/67333662/1ca23d91-d690-4640-99a0-5efebc22c136)